nomad-base/src/indexer.rs

```diff
@@ -162,8 +162,10 @@ impl HomeIndexer for HomeIndexerVariants {
     async fn fetch_sorted_messages(&self, from: u32, to: u32) -> Result<Vec<RawCommittedMessage>> {
         match self {
             HomeIndexerVariants::Ethereum(indexer) => indexer.fetch_sorted_messages(from, to).await,
+            HomeIndexerVariants::Substrate(indexer) => {
```

Comment:
Nit: Bracket use

Reply:
These are all decided by `fmt` not me :)